LICENSE

```diff
@@ -0,0 +1,5 @@
+Couchbase Server Documentation © 2024 by Couchbase Inc. is licensed under Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International 
+
+CC BY-NC-SA 4.0.
+
+See https://creativecommons.org/licenses/by-nc-sa/4.0/ for details.
```

Comment:
Note, in the SDKs we've gone for presenting the complete licence, to save users a further click (away from the site):
https://github.com/couchbase/docs-sdk-scala/blob/release/1.5/LICENSE

And the new IA SDK docs (coming in Q2) have a mention in the README - https://github.com/couchbase/docs-sdk-scala/blob/release/1.6/README.adoc#docs-license

Reply:
I didn't see the SDK docs had the license file already. Searching through github, it does seem common for repos by major companies (i.e. Microsoft) to include the full license text in their LICENSE file, so it makes sense to follow suit.

Reply:
Well, if hipness is our goal, we could link to a TikTok video of someone doing an interpretive dance of the CC license text.

Reply:
I second the motion.

Reply:
This is a proposal we can all get behind. Let's set a date for the next docs summit.